src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/Microsoft.DotNet.SourceBuild.SmokeTests.csproj

```
@@ -12,6 +12,10 @@
     <VSTestUseMSBuildOutput>false</VSTestUseMSBuildOutput>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="..\..\eng\tools\tasks\Microsoft.DotNet.UnifiedBuild.Tasks\Utilities.cs" />
```

nit: I would use TasksDir here

```suggestion
    <Compile Include=$(TasksDir)Microsoft.DotNet.UnifiedBuild.Tasks\Utilities.cs" />
```